src/main/resources/static/js/testrules.js

```diff
@@ -364,6 +364,40 @@ jQuery(document).ready(
         event.preventDefault();
         getTemplate("eventsTemplate");
       });
+      
+   // Start to check is backend Test Rule service status
+    	var isEnabled = true;
+    	console.log('Here are my modifications');
+    	$.ajax({
+    		url: frontendServiceUrl + "/rules/rule-check/testRulePageEnabled",
+    		contentType : 'application/json; charset=utf-8',
+    		type: 'GET',
+    		error: function () {},
+    		success: function (data) {
+    			isEnabled = JSON.parse(ko.toJSON(data)).status;
+    			if(isEnabled == false) {
+    				displayOverlay("Test Rule service is not enabled! To enable it set the backend property [testaggregated.enabled] as [true]")}
+    			},
+    		complete: function () { }
+    	});
+    	// Finish to check backend Test Rule Service status
+    	
+    	function displayOverlay(text) {
+    	    $("<table id='overlay'><tbody><tr><td>" + text + "</td></tr></tbody></table>").css({
```

Comment:
Is there no other place where we can put the CSS code? If not i guess this would be fine however I really think we should have files dedicated to CSS so that its easier to change the looks.